org.kuchelmeister.ToolboxTuner.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id" : "org.kuchelmeister.ToolboxTuner.Devel",
```

Comment:
```suggestion
    "id" : "org.kuchelmeister.ToolboxTuner.Devel",
```

this doesn't match. `Devel` isn't a proper app name.

Reply:
This was an accidental copy paste leftover from the development config. I fixed it in 77bcfab7c6b